corehq/apps/case_search/tests/test_models.py

```diff
@@ -85,15 +84,10 @@ def test_extract_criteria_config(self, case_type, data_registry, custom_related_
 def _make_request_dict(params):
     """All values must be a list to match what we get from Django during a request.
     """
-    request_dict = MultiValueDict()
-    for key, value in params.items():
-        if value is None:
-            continue
-        if isinstance(value, list):
-            request_dict.setlist(key, value)
-        else:
-            request_dict[key] = value
-    return request_dict
+    return {
+        key: (value if isinstance(value, list) else [value])
+        for key, value in params.items() if value is not None
+    }
```

Comment:
do none of the uses rely on any of the methods of `MultiValueDict` e.g. `getlist`?

Reply:
Nope, it's only used in one place, which converts it to a normal dict.  I moved that coercion up the stack to make it easier to initialize in other contexts.
https://github.com/dimagi/commcare-hq/blob/ef722b033b9dfe1dc15b4129915806a7152ea183/corehq/apps/case_search/models.py#L199-L200